src/st/wrp_psp/e_collect.c

```diff
@@ -72,13 +72,13 @@ void CollectHeart(u16 arg0) {
 }
 
 extern u8* D_80180E08[];
-extern s32 c_GoldPrizes[];
+extern u32 c_GoldPrizes[];
 
 void CollectGold(u16 goldSize) {
```

Comment:
I'm thinking Xeeynamo might want the PSP version de-duplicated too? https://github.com/Xeeynamo/sotn-decomp/pull/1208#pullrequestreview-2086627080

Reply:
Sounds good, done